Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+
+gem 'fpm'
```

Comment:
yeah if nothing uses the Gemfile then we shouldn't have it, because it just makes things confusing.